.github/workflows/foreman.yml

```diff
@@ -122,6 +122,11 @@ jobs:
         run: |
           bundle exec rake db:create
           bundle exec rake db:migrate
+      - name: Archive all_react_app_exports
```

Comment:
You can set
```
if: contains(matrix.task, 'compile')
```
as that's the only task where we compile webpack.

Reply:
oh great, I thought it would be more complicated, thanks!